src/System.Net.Security/tests/FunctionalTests/SslStreamNetworkStreamTest.cs

```diff
@@ -101,11 +101,8 @@ public class SslStreamNetworkStreamTest
                 // Initiate Read operation, that results in starting renegotiation as per server response to the above request.
                 int bytesRead = await ssl.ReadAsync(message, 0, message.Length);
 
-                // SslStream.ReadAsync should return -1 bytes, indicating renegotiation operation is in progress.
-                Assert.Equal(-1, bytesRead);
-
-                // Do another Read, to get the HTTP response from the server, after successful renegotiation.
-                bytesRead = await ssl.ReadAsync(message, 0, message.Length);
+                // There's no good way to ensure renegotiation happened in the test.
+                // Under the debugger, we can see this test hits the renegotiation codepath.
```

Comment:
I assume it shows in code coverage reports as well?

Reply:
Yup,
![image](https://user-images.githubusercontent.com/7853870/35164867-f54d67b4-fd00-11e7-8d99-f6453aa7e357.png)

Reply:
Great, thanks.